githubscript/docker.d/worker.yml

```diff
@@ -20,4 +20,10 @@ github_projects:
         github_token: dummy
         github_owner: non-existing-owner
         github_repo_name: non-existing-repo
+    'COT_PRODUCT == "mobile" && ENV == "dev"':
+      mock:
+        allowed_actions: ["release"]
+        github_token: dummy
```

Comment:
We could use `{ "$eval": "GITHUB_TOKEN_WRITE_ACCESS_STAGING" }` here. This way, we actually exercise code that uploads something onto a github repo. What do you think?